### ISSUE TITLE
Feature Request: XML option to force re-download of mod

### ISSUE THREAD

**Issue by Cherry** [ATLauncher/ATLauncher#216]
Currently, if a file with the same name already exists in the user's downloads folder, the only way to force a re-download of a mod is to change the md5. When hosting resource packs externally that get frequently updated, it'd be nice to be able to strip the md5, but use a flag that forces a re-download every install.

Happy to provide any further info if necessary!

**Comment by Divineaspect** [comment]
That would also help for a few people who do "stealth" updates without changing version numbers, like Recurrent Complex this week.

**Comment by RyanTheAllmighty** [comment]
This will be added as an attribute to the <mod /> tag as:

```
<mod force="true" />
```

With the default value being false.

When force is true, for any mod, it will discard any file already present which will force a redownload once per install.

This won't make it's way into the launcher until version 3.2.3.2 which at that time I'll close this issue so you'll be notified.

But in the mean time I've made it valid XML, it just won't do anything at the moment :)

**Comment by RyanTheAllmighty** [comment]
Heck, while I'm here I'll pop it on the libraries also, cause who knows

**Comment by Cherry** [comment]
Awesome, thanks!

**Comment by RyanTheAllmighty** [comment]
Alright it's now in the XML -> JSON conversion system so just waiting on the launcher update

**Comment by leagris** [comment]
Thank you @RyanTheAllmighty about #228 review.

**Comment by RyanTheAllmighty** [comment]
Okay this is added and will be released with 3.2.3.2

**Comment by RyanTheAllmighty** [comment]
Released
